assets/component-cart-items.css

```diff
@@ -323,3 +323,17 @@ cart-remove-button .icon-remove {
     width: 60%;
   }
```

Comment:
I wonder if we should hide the info button for no js because it only works by clicking on the input https://screenshot.click/05-14-ldkv2-c3x16.mp4

Reply:
Updated - https://github.com/Shopify/dawn/pull/2771/commits/a6967944ebf0bb6b091646da7b45a6acbb104f23